src/pluginHooks.js

```diff
@@ -1460,9 +1463,60 @@ const REGISTERED_HOOKS = [
    * ```
    */
   'afterFilter',
-
   /* eslint-enable jsdoc/require-description-complete-sentence */
 
+  /**
+   * Called when a value is updated in the engine.
+   *
+   * @event Hooks#afterFormulasValuesUpdate
+   * @param {Array} changes The values and location of applied changes.
```

Comment:
Should we add `@since 9.0.0` tag for each new hook?

Reply:
https://github.com/handsontable/handsontable/pull/7648/commits/aa54313fdb5882f40f8e5adfc81514133c9acbe7